web/src/search/input/interactive/FilterInput.test.tsx

```diff
@@ -0,0 +1,143 @@
+import React from 'react'
+import { FilterInput } from './FilterInput'
+import { FilterTypes, FiltersToTypeAndValue } from '../../../../../shared/src/search/interactive/util'
+import sinon from 'sinon'
+import { render, fireEvent, cleanup, getByText, getByDisplayValue } from '@testing-library/react'
+
+const defaultFiltersInQuery: FiltersToTypeAndValue = {
+    fork: {
+        type: FilterTypes.fork,
+        value: 'no',
+        editable: false,
+        negated: false,
+    },
+}
+const defaultProps = {
+    filtersInQuery: defaultFiltersInQuery,
+    navbarQuery: { query: 'test', cursorPosition: 4 },
+    mapKey: 'repo',
+    value: '',
+    filterType: FilterTypes.repo,
+    editable: true,
+    negated: false,
+    isHomepage: false,
+    onSubmit: sinon.spy(),
+    onFilterEdited: sinon.spy(),
+    onFilterDeleted: sinon.spy(),
+    toggleFilterEditable: sinon.spy(),
+    toggleFilterNegated: sinon.spy(),
+}
+
+describe('FilterInput', () => {
+    afterAll(cleanup)
+    let container: HTMLElement
+    beforeEach(() => {
+        ;({ container } = render(<FilterInput {...defaultProps} editable={true} />))
```

Comment:
Nit: I personally prefer the following:

```suggestion
        container = render(<FilterInput {...defaultProps} editable={true} />).container
```

 I found the parentheses with the inserted semicolon a bit distracting...

Additionally, since you seem to start each test with a `render()`, do you need this `beforeEach()`?